spell/spell-crafter-mainnet-workflow.md

```diff
@@ -123,13 +124,15 @@ Repo: https://github.com/makerdao/spells-mainnet
       * Patch -> Collateral addition or addition/modification (0.0.++)
     * [ ] New addresses are added to the `addresses_mainnet.sol`
     * [ ] Changes are tested via `testNewOrUpdatedChainlogValues`
+    * [ ] Additions are tested via `testAddedChainlogKeys`
```

Comment:
Actually this replaces the check above. You can remove it.

Reply:
Addressed in [`ddea55c`](https://github.com/sky-ecosystem/pe-checklists/pull/52/commits/ddea55c29bc22eddb99004f10246aecf8ef4b9c7)